src/components/Transaction/Total.tsx

```diff
@@ -15,13 +15,13 @@ const Total = styled.div`
 
 export const TransactionTotal = ({ total }: TransactionTotalProps) => {
   return (
-    <footer>
+    <>
```

Comment:
Removing the footer container makes the style more consistent now it's contained in the `Lines` wrapper

Reply:
So nice!